src/site/content/en/blog/iframe-lazy-loading/index.md

```diff
@@ -295,6 +295,24 @@ lazysizes when it's not available:
 </script>
 ```
 
+### An option for Wordpress users {: #wordpress }
+
+You might have many iframes scattered across years worth of post content
+in a Wordpress site. You can optionally add the following code to your Wordpress
+theme's `functions.php` file to automatically insert `loading="lazy"` to your
+existing iframes without having to manually update them each individually.
```

Comment:
```suggestion
existing iframes without having to manually update them each individually.

Note that [native support for lazy-loading iframes is also being worked on in WordPress core](https://core.trac.wordpress.org/ticket/50756).
The following snippet will check for the relevant flags so that, once WordPress has the
functionality built-in, it will no longer manually add the `loading="lazy"` attribute,
ensuring it is interoperable with those changes and will not result in a duplicate attribute.
```

Reply:
I'll add a comment to the PHP code